src/actions/advancement.cpp

```diff
@@ -194,10 +194,10 @@ namespace
 		{
 			//the 'side' parameter might differ from side_num_-
 			int res = 0;
-			team t = resources::gameboard->get_team(side_num_);
+			const team t = resources::gameboard->get_team(side_num_);
```

Comment:
not really related, but this seems wrong since it copies the whole `team` object